frontend/packages/console-dynamic-plugin-sdk/src/app/core/reducers/index.ts

```diff
@@ -0,0 +1,2 @@
+export { coreReducer } from './core';
+export * from './coreSelector';
```

Comment:
```suggestion
export * from './coreSelectors';
```
Singular denotes one like a default export and some additional items... 

Just like it's called `utils` and not `util` when making some utility file.